frontend_app_admin_workspace_user/test/component/AddUserForm.spec.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow } from 'enzyme'
+import { AddUserForm } from '../../src/component/AddUserForm.jsx'
+
+describe('<AddUserForm />', () => {
+  const props = {
+    emailNotifActivated: true,
+    isEmailRequired: true,
+    isUsernameValid: false
+  }
+
+  const wrapper = shallow(<AddUserForm {...props} t={tradKey => tradKey} />)
+
+  describe('its internal functions', () => {
+    describe('isValidateButtonDisabled', () => {
+      it('should return true if the email notification are activated, the email and the password are empty', () => {
+        wrapper.setState({ newUserEmail: '' })
+        wrapper.setState({ newUserPassword: '' })
+        wrapper.setState({ newUserName: '' })
+        wrapper.setState({ newUserProfile: '' })
+        wrapper.setState({ newUserUsername: '' })
+        expect(wrapper.instance().isValidateButtonDisabled()).to.equal(true)
+      })
+
+      it('should return true if the name or the profile are empty', () => {
```

Comment:
it seems that this test expect empty values that are set in the previous test.
This make it looks like tests aren't independent.
If I'm correct, you should put some `wrapper.setState` of the previous test in a `before()` function

Reply:
changed